test/query_parser/test_parser.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from src.query_parser.eva_parser import EvaFrameQLParser
+from src.query_parser.eva_statement import EvaStatement
+from src.query_parser.eva_statement import StatementType
+from src.query_parser.select_statement import SelectStatement
+from src.expression.abstract_expression import ExpressionType
+from src.query_parser.table_ref import TableRef
+
+
+class ParserTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Can we also add some test cases for scenarios where the attributes and name of video are in lower case? 

In SQL, it's common to have Keywords(SELECT, FROM, JOINT etc.) as capitals and everything else in lowercase.